src/SearchResults.js

```diff
@@ -0,0 +1,47 @@
+//creating a table
+import React from "react";
+import moment from "moment";
+
+const SearchResults = props => {
+  return (
+    <div>
+      <table class="table table-bordered table-dark">
```

Comment:
Good to see you're using bootstrap - it's a really useful and powerful library so definitely a good thing to get to grips with.